yarn.lock

```diff
@@ -3659,7 +3675,7 @@ typescript@^5.2.2, typescript@^5.3.3:
     "@myriaddreamin/typst-ts-renderer" "0.5.0-rc5"
     "@myriaddreamin/typst.ts" "0.5.0-rc5"
     rxjs "^7.8.1"
-    typst-dom "link:../../../AppData/Local/Yarn/Cache/v6/npm-typst-preview-frontend-0.0.0-e4b78602-ab80-42c0-9a76-2c66cb729491-1719528135321/node_modules/typst-dom"
```

Comment:
we can discard changes of yarn.lock.

Reply:
I'll remove this change in the next commit